### DISCUSSION TITLE
Any Genre 1 Location field unpopulated in Connect dialog

### DISCUSSION THREAD

**Discussion by rdica** [jamulussoftware/jamulus#2978] [Technical Support & Help] [answered]
I noticed Thursday that the Location field in the Connect dialog doesn't show any data for servers in Any Genre1, has something changed?

**Comment by dcorson-ticino-com** [comment]
All is showing for me using 3.9.1 this morning.

**Comment by pljones** [comment, marked as answer]
Maybe you're only getting the fast response list through and the full list is getting broken up somewhere between the directory and your client, preventing it's use.  (I'm never quite sure which fields are in the cut down fast response list but it's meant to be just enough that the client can do what it needs to do.)

As always, [Jamulus Explorer](https://explorer.jamulus.io) is a useful diagnostic tool: it uses the same protocol requests as a real client to retrieve information from the directories.

**Reply by rdica** [reply]
Yeah i confirmed there before posting and saw location info. Never saw this behaviour before in my client, and nothing's changed with my connection to ISP or local network, must be something upstream :(

**Reply by rdica** [reply]
Nothing has changed on my end but the location field data has returned in the client. Calling this closed.
